core/examples/http_bench.js

```diff
@@ -1,11 +1,77 @@
 // This is not a real HTTP server. We read blindly one time into 'requestBuf',
 // then write this fixed 'responseBuf'. The point of this benchmark is to
 // exercise the event loop in a simple yet semi-realistic way.
-const OP_LISTEN = 1;
-const OP_ACCEPT = 2;
-const OP_READ = 3;
-const OP_WRITE = 4;
-const OP_CLOSE = 5;
+const registry = {};
```

Comment:
I don't like that this registry map is a second source of truth for the op maps. (The real one lives in the "OpRegistry" in Rust.)
Can you remove this map entirely?

This is making me think maybe we need an op for adding ops.

Reply:
It's not really a source of truth - Rust is source of truth in this case. This structure is used only to store all js ops in single place and set ops ids when they arrive from Rust on initialization. 

> This is making me think maybe we need an op for adding ops.

Can you elaborate?

Reply:
I think this registry map could be improved by using a Map type instead of an object and use js doc to label the type of key/values as @kitsonk has done [here](https://github.com/denoland/deno/blob/3d2d0ee771abe308aee9a0ab6c89fd09bac80330/deno_typescript/compiler_main.js#L112-L115).

>> This is making me think maybe we need an op for adding ops.

> Can you elaborate?

I'm not sure about it...  I will think about it more after we get this one landed.